DB_class/DB_config.py

```diff
@@ -110,7 +110,24 @@ def select_by_rating(self, str_input):
             player_id_list.append(row[0])
         return player_id_list
 
+    def select_item_id(self):
+        item_id_list = []
+        query = """SELECT DISTINCT itemID from itemTbl;"""
+        self.curs.execute(query)
+        rows = self.curs.fetchall()
+        for row in rows:
```

Comment:
This is a minor problem, But the variable name(rows, row) can caused human error
plz change it more clearly

Reply:
17abc1ae09297390190115b92a5dd549fbaed78d